city_scrapers/spiders/il_aging_advisory_council.py

```diff
@@ -0,0 +1,119 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlAgingAdvisoryCouncilSpider(CityScrapersSpider):
+    name = "il_aging_advisory_council"
+    agency = "Illinois Aging Advisory Council"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www2.illinois.gov/aging/PartnersProviders/OlderAdult/Pages/acmeetings.aspx"
+    ]
+    location = {
+        'address': '160 N. LaSalle Street, 7th Floor, Chicago',
+        'name': 'Michael A. Bilandic Building'
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+
+        self._validate_meeting_times(response)
+
+        table = response.xpath(
+            '//*[@id="ctl00_PlaceHolderMain_ctl01__ControlWrapper_RichHtmlField"]\
+            /table'
+        )
+
+        for table_row in table.xpath('.//tr'):
+            row_date = table_row.xpath('.//strong/\
+            text()').re(r'([A-Z]\w+)\s*(\d\d?),*\s*(\d\d\d\d)')
+            if not row_date:
+                continue
+            else:
```

Comment:
Since none of the code after the if condition will run if it's not true, you can remove this `else` and just have `converted_date` be assigned on the next line outside the condition